staff/views/users.py

```diff
@@ -84,7 +81,7 @@ def get_queryset(self):
         if types := self.request.GET.getlist("types"):
             qs = qs.filter(type__in=types)
 
-        return fetch(qs.distinct())
+        return list(qs.distinct())
```

Comment:
If we return a queryset (i.e. remove `list`), then we actually _reduce_ the number of queries from 14 to 13.

Reply:
Good spot. That's the case for both of them infact.